PowerEditor/src/NppCommands.cpp

```diff
@@ -487,7 +487,7 @@ void Notepad_plus::command(int id)
 			if (nppGui._searchEngineChoice == nppGui.se_custom)
 			{
 				url = nppGui._searchEngineCustom;
-				remove_if(url.begin(), url.end(), _istspace);
+				static_cast<void>(std::remove_if(url.begin(), url.end(), _istspace));

```

Comment:
Visual Studio 2019 refused to compile due to `remove_if` being annotated as `nodiscard`.

Reply:
If you're using VS2019, you should build using VS2017 vc141 toolset.
See more info here: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9922

Reply:
In lieu of installing the 141 toolset, why not just use 142 by default?